persistent/Database/Persist/Sql/Types.hs

```diff
@@ -34,6 +34,7 @@ data Column = Column
 -- | This value specifies how a field references another table.
 --
 -- @since 2.11.0.0
+-- TODO: what about schema name there?
```

Comment:
This does seem important to resolve. Unfortunately, the `ColumnReference` type is exposed, meaning that we will incur a major version bump if we modify it.